addons/better-img-uploads/userscript.js

```diff
@@ -0,0 +1,200 @@
+export default async function ({ addon, console, safeMsg: m }) {
+  let mode = addon.settings.get("fitting");
+
+  addon.settings.addEventListener("change", () => {
+    mode = addon.settings.get("fitting");
+  });
+
+  //Returns html code for an item in the selection lists, complete with tooltip.
+  let html = (id, right) => `<div id="${id}">
+  <button aria-label="Upload Costume" class="${addon.tab.scratchClass("action-menu_button")} ${addon.tab.scratchClass(
+    "action-menu_more-button"
+  )} sa-better-img-uploads-btn" data-for="sa-${id}-HD Upload" data-tip="${m("upload")}" currentitem="false">
+    <img class="${addon.tab.scratchClass("action-menu_more-icon")} sa-better-img-uploader" draggable="false" src="${
+    addon.self.dir + "/icon.svg"
+  }" height="10", width="10">
+     <input accept=".svg, .png, .bmp, .jpg, .jpeg, .gif" class="${addon.tab.scratchClass(
+       "action-menu_file-input"
+     )}" multiple="" type="file">
+  </button>
+  <div class="__react_component_tooltip place-${right ? "left" : "right"} type-dark ${addon.tab.scratchClass(
+    "action-menu_tooltip"
+  )} sa-better-img-uploads-tooltip" id="sa-${id}-HD Upload" data-id="tooltip" >${m("upload")}</div>
+</div>`;
+
+  while (true) {
+    //Catch all upload menus as they are created
+    let menu = await addon.tab.waitForElement('[class^="action-menu_more-buttons"]', { markAsSeen: true });
+    let button = menu.parentElement.previousElementSibling.previousElementSibling; //The base button that the popup menu is from
```

Comment:
It's not working for me now because button is null

Reply:
Oh. It needs to be
```js
let menu = await addon.tab.waitForElement('[class*="action-menu_more-buttons"]', { markAsSeen: true });
```